flang/include/flang/Runtime/allocatable.h

```diff
@@ -1,4 +1,4 @@
-//===-- runtime/allocatable.h -----------------------------------*- C++ -*-===//
+//===-- include/flang/Runtime/allocatable.h ----------------
```

Comment:
This file contains C++ code. Why remove the `-*- C++ -*-===//`?

Also, of the 1309 .h files in `llvm/include/llvm`, there is only 1 file that includes the `include/llvm` path prefix. The convention would be `Runtime/allocatable.h`.